mautic/mautic
# Add analytics to email preview

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Add script to email preview. Already we can use scripts for landing page and webview email.
Butt sometimes we use email preview as click link to web notifications or another channel. 
Prevent insert scripts code if user is logged in Mautic

#### Steps to test this PR:
1.  Apply PR and remove cache app/cache/prod/
2. Create email and open email preview link in inkognito mode (or another browser)
5. See source code with analytics code in the header
